app/controllers/portal/client_logins_controller.rb

```diff
@@ -63,7 +63,9 @@ def check_verification_code
       render :enter_verification_code
     end
 
-    def invalid_token; end
+    def invalid_token
+      redirect_to portal_client_logins_path
+    end
```

Comment:
It looks like we could probably delete this action in a followup deploy, since we'll no longer be redirecting to it.

Reply:
I think it's good to deploy a deletion separately, in the small chance that someone has been redirected to this page right before this deploy.